onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -345,7 +345,8 @@ def aten_all(self: TTensor) -> BOOL:
     else:
         self_bool = op.Cast(self, to=BOOL.dtype)
         self_int = op.Cast(self_bool, to=INT64.dtype)
-        all_true = op.ReduceMin(self_int, keepdims=False)
+        empty_axes = op.Shape(self, start=0, end=0)
+        all_true = op.ReduceMin(self_int, empty_axes, keepdims=False)
```

Comment:
is empty_axes just `[]`?

Reply:
That what I had initially, but this error is raised,
` onnxscript.converter.TranslationError: ERROR: Cannot convert an empty list to tensor
E   at: Function 'elseGraph_5', line 12
E           any_true = op.ReduceMax(self_int, axes=[], keepdims=False)`

Reply:
We can't use `[]` because the translator cannot figure out the intended type of the tensor.

Nit: I wonder if `op.Constant(value_ints = [])` (IF IT WORKS) will be more natural then using Shape op.

Reply:
On an unrelated note: why is it cast to `INT64` in line 347? Wouldn't `INT32` or `INT16` work just as well, while being more efficient?

Reply:
> op.Constant(value_ints = []

Yup, this will work, and better alternative to the Shape op.